server/src/sim/makeccocs.py

```diff
@@ -428,14 +431,13 @@ def gettargetpop(D=None, artindex=None, progname=None):
                     targetpopmodel = multiply(D['M']['birth'][:,0:npts], D['M']['breast'][0:npts], D['S']['people'][artindex,:,0:npts].sum(axis=0)).sum(axis=0)
                 elif thispar in ['numfirstline','numsecondline']: # Target population = diagnosed PLHIV
                     targetpopmodel = D['S']['people'][artindex,:,0:npts].sum(axis=(0,1))
-                else:
-                    print('WARNING, Unrecognized parameter %s' % thispar)
+                elif thispar == 'numcircum': # Target population = men (??)
+                    maleindices = [i for i, x in enumerate(D['data']['meta']['pops']['female']) if x == 0]
```

Comment:
rather than comparing female to 0, shouldn't you be comparing male to 1?
D['data']['meta']['pops']['male'] == 1
some populations can be neither or both male and female